integration_tests/mockApis/manageUsersApi.ts

```diff
@@ -0,0 +1,43 @@
+import { Response } from 'superagent'
+
+import { stubFor } from './wiremock'
+
+const stubUser = (name: string) =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: '/manage-users-api/users/me',
+    },
+    response: {
+      status: 200,
+      headers: {
+        'Content-Type': 'application/json;charset=UTF-8',
+      },
+      jsonBody: {
+        staffId: 231232,
+        username: 'USER1',
+        active: true,
+        name,
+      },
+    },
+  })
+
+const stubUserRoles = () =>
```

Comment:
the old `stubUserRoles` used to allow passing in an optional list of roles, but this one does not.

it looks like there's now [a test](https://github.com/ministryofjustice/hmpps-incentives-ui/pull/646/files#diff-db80e94f69111242b7dbaf1b18f51d011296fa55b823d80c66e948af17d54deaR12-R15) that is using `stubSignIn` & `stubUserRoles` incorrectly: it passes override roles to `stubSignIn` (which doesn't accept them)

Reply:
found [another test too](https://github.com/ministryofjustice/hmpps-incentives-ui/pull/646/files#diff-db80e94f69111242b7dbaf1b18f51d011296fa55b823d80c66e948af17d54deaR12-R17)

Reply:
Done here, hope this is what you had in mind: https://github.com/ministryofjustice/hmpps-incentives-ui/pull/646/commits/17db7d789466101f5b6674ebb30a9a21e5986032

Reply:
yup! i was wrong about `stubSignIn` (i was looking at a different method) but am happy with the new `stubUserRoles`!